src/filter/ruleComponent.js

```diff
@@ -424,24 +424,21 @@ exports.RuleController_ = class {
       this.unlisteners_.push(this.scope_.$watch(
         () => this.clone.getExpression(),
         (newVal) => {
-          const value = newVal === null ? this.createDate_() : newVal;
-          this.timeValueMode.minValue = value;
+          this.timeValueMode.minValue = newVal;
```

Comment:
For that is this code for?

Reply:
Not idea.
But since we now handle the null value correctly I suppose the ternary operation can be removed.